beetsplug/acoustic.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import (division, absolute_import, print_function,
```

Comment:
This will (eventually) need the standard comment header found at the top of all the files in the tree.